x/gamm/keeper/swap_test.go

```diff
@@ -210,20 +210,26 @@ func (suite *KeeperTestSuite) TestCalcOutAmtGivenIn() {
 			suite.SetupTest()
 			keeper := suite.App.GAMMKeeper
 			ctx := suite.Ctx
-
 			var pool poolmanagertypes.PoolI
 			if test.param.poolType == "balancer" {
 				poolId := suite.PrepareBalancerPool()
 				poolExt, err := suite.App.GAMMKeeper.GetPool(suite.Ctx, poolId)
 				suite.NoError(err)
-				pool = poolExt.(poolmanagertypes.PoolI)
+				poolI, ok := poolExt.(poolmanagertypes.PoolI)
```

Comment:
why not use `pool` variable here directly?

Reply:
same question for other similar `pool = poolI` cases

Reply:
so, the "type assertion" -- which I had to spend a bunch of time researching today, allows you to get at the "underlying type" of the variable, and those are set up like this:

```
poolExt.(poolmanagertypes.PoolI)
```

but they can fail.  But... as for... why not just call it pool -- in fact, maybe we should :)